packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx

```diff
@@ -44,10 +44,11 @@ export const registerServiceWorker = (url) => {
 export const OuterApp = ({routes, error, WrappedApp, locals, onHydrate}) => {
     const AppConfig = getAppConfig()
     const isInitialPageRef = useRef(true)
+    const basepath = AppConfig.getBasePath?.({req: undefined, res: undefined}) || ''
```

Comment:
Are you forgetting to chop off the trailing slash as you do [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1988/files#diff-ce5d18a6f177da815bcff05c82ba4c307d2bbcf90c1645561f63598dea797495R149)
?